### DISCUSSION TITLE
user account is blocked

### DISCUSSION THREAD

**Discussion by VerdeVistaVoyager** [vercel/community#4007] [Account Issue] [answered]
### Question

Hello, I would like to sign up to try out vercel. My github is: VerdeVistaVoyager

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by avilyre** [reply]
I have the same error with my account avilyrs@gmail.com @amyegan
